tests/Support/SupportHelpersTest.php

```diff
@@ -315,6 +315,21 @@ public function testStrLimit()
         $this->assertEquals('这是一', Str::limit($nonAsciiString, 6, ''));
     }
 
+    public function testStrWords()
+    {
+        $string = 'Hello world!';
+        $this->assertEquals('Hello...', str_words($string, 1));
+        $this->assertEquals('Hello___', str_words($string, 1, '___'));
+        $this->assertEquals('Hello', str_words($string, 1, ''));
+        $this->assertEquals('Hello world!', str_words($string, 3));
+
+        $unicodeString = 'Привет мир!';
+        $this->assertEquals('Привет...', str_words($unicodeString, 1));
+        $this->assertEquals('Привет___', str_words($unicodeString, 1, '___'));
+        $this->assertEquals('Привет', str_words($unicodeString, 1, ''));
+        $this->assertEquals('Привет мир!', str_words($unicodeString, 3));
```

Comment:
As you can see i in the other Str tests, its the classs that is used not the helper :)

Reply:
@stefanoruth The `Str` tests belongs to `SupportStrTest` class. Yup, things got mixed somehow.